Makefile.kokkos

```diff
@@ -272,6 +272,8 @@ ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
     KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fiopenmp -Wno-openmp-mapping
   else ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
     KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -mp=gpu 
+  else ifeq ($(KOKKOS_INTERNAL_COMPILER_CLANG), 1)
+    KOKKOS_INTERNAL_OPENMPTARGET_FLAG := -fopenmp -Xopenmp-target
```

Comment:
Why are you modifying `OpenMPTarget` flags here?

Reply:
I agree that this change is unrelated to the OpenACC backend; I added that since we can compile the OpenMPTarget backend using the public Clang/LLVM but not included in the Makefile.
Or, I can delete this and submit it as a separate PR.

Reply:
@rgayatri23: you are right; the flag was wrong (and redundant). Therefore, this flag is deleted.